tests/mcpdotnet.Tests/ClientIntegrationTests.cs

```diff
@@ -382,4 +405,69 @@ public async Task CallTool_Stdio_MemoryServer()
 
         await client.DisposeAsync();
     }
+
+    [Fact]
+    public async Task GetAIFunctionsAsync_UsingEverythingServer_ToolsAreProperlyCalled()
+    {
+        if (s_openAIKey is null)
+        {
+            return; // Skip the test if the OpenAI key is not provided
+        }
+
+        // Get the MCP client and tools from it.
+        await using var client = await McpClientFactory.CreateAsync(_fixture.EverythingServerConfig, _fixture.DefaultOptions); ;
+        var mappedTools = await client.GetAIFunctionsAsync();
+
+        // Create the chat client.
+        using IChatClient chatClient = new OpenAIClient(s_openAIKey).AsChatClient("gpt-4o-mini")
```

Comment:
Should we use OllamaSharp to make it easier for other contributors to run our integration tests?

Reply:
Maybe, but Ollama takes away some complication and adds other complication.